### DISCUSSION TITLE
Awesome Privacy helped inspire PrivacyShortlist.com: the best privacy-focussed tools for *startups*

### DISCUSSION THREAD

**Discussion by fredrivett** [pluja/awesome-privacy#259] [Show and tell]
Hey folks,

I love what's been built here with Awesome Privacy. The level of detail and breadth of number of categories and tools is ace!

That said, as someone who wanted to find privacy-focussed tools to build my next **startup**, quite a lot of the tools here seemed aimed at individual consumers, so I decided to spend 89h 6m building a little site just for that use case.

![image](https://user-images.githubusercontent.com/1713859/194045571-eab39c99-9661-4350-86cf-a1c3f0435e02.png)

It's clearly MVP (see time spent so far), but enables you to login with twitter or one click email auth, to show some love to your favourite products.

I'd love any honest thoughts and feedback you've got on the project and how I can make it better 👇

[📝 PrivacyShortlist.com](https://privacyshortlist.com/)

Thanks folks! ❤️

p.s. Hopefully `🙌 Show and tell` is the right place for this sort of post, if not apologies!

**Comment by pluja** [comment]
Hey! That's an awesome work! I'm happy this list inspired you :)

**Reply by fredrivett** [reply]
Ah great to hear from you @pluja, thanks so much for taking a look and the kind words.

It's still very early days, just working out how best to get the word out so that more folks are aware of the resource and can choose a privacy-focussed tool next time. Definitely open to suggestions if you have any!
